templates/Export/default_template.tpl

```diff
@@ -283,12 +283,9 @@ Ob es sich um den Verzicht auf Erstattung von Aufwendungen handelt, ist der Anla
 </div>
 
 <div class="footer">
-    <p><strong>Hinweis:</strong><br />Hinweis: Wer vorsätzlich oder grob fahrlässig eine unrichtige Zuwendungsbestätigung erstellt oder wer veranlasst, dass Zuwendungen nicht
-zu den in der Zuwendungsbestätigung angegebenen steuerbegünstigten Zwecken verwendet werden, haftet für die Steuer, die dem Fiskus
-durch einen etwaigen Abzug der Zuwendungen beim Zuwendenden entgeht (§ 10b Abs. 4 EStG, § 9 Abs. 3 KStG, § 9 Nr. 5 GewStG).
-Diese Bestätigung wird nicht als Nachweis für die steuerliche Berücksichtigung der Zuwendung anerkannt, wenn das Datum des
-Freistellungsbescheides länger als 5 Jahre bzw. das Datum der vorläufigen Bescheinigung länger als 3 Jahre seit Ausstellung der Bestätigung
-zurückliegt (BMF vom 15.12.1994 - BStBl I S. 884).</p>
+    <p><strong>Hinweis:</strong><br /> Wer vorsätzlich oder grob fahrlässig eine unrichtige Zuwendungsbestätigung erstellt oder veranlasst, dass Zuwendungen nicht zu den in der Zuwendungsbestätigung angegebenen steuerbegünstigten Zwecken verwendet werden, haftet für die entgangene Steuer (§ 10b Abs. 4 EStG, §9 Abs. 3 KStG, § 9 Nr. 5 GewStG).
```

Comment:
I prefer to put non-breaking spaces between `§` signs (Sect.) or `Abs.` (para.) and the corresponding paragraph/ number, so that no line break occurs between them: `§ 10b Abs. 4 EStG` would then read `§&nbsp;10b Abs.&nbsp;4&nbsp;EStG` - but I think that's just cosmetics.

Reply:
Also, can we have a maximum line length (I think 80 characters would be ok) and proper indentation?